olive/workflows/run/__main__.py

```diff
@@ -3,21 +3,39 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 import argparse
+import sys
+from pathlib import Path
 
 from olive.common.utils import set_tempdir
 from olive.workflows import run
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser("Olive Workflow: Custom Run")
-    parser.add_argument("--config", type=str, help="Path to json config file", required=True)
+    parser.add_argument("--olive-config", type=str, help="Path to Olive (json) config file", required=False)
+    parser.add_argument("--run-config", "--config", type=str, help="Path to json config file", required=True)
     parser.add_argument("--setup", help="Whether run environment setup", action="store_true")
-    parser.add_argument("--data_root", help="The data root path for optimization", required=False)
+    parser.add_argument("--data-root", "--data_root", help="The data root path for optimization", required=False)
     parser.add_argument("--tempdir", type=str, help="Root directory for tempfile directories and files", required=False)
 
     args = parser.parse_args()
 
+    if "--config" in sys.argv:
+        print(  # noqa:T201
+            "WARNING: Option '--config' is deprecated and will be removed in future versions. "
+            "Use '--run-config' instead."
+        )
```

Comment:
We don't want to deprecate --config.
Is there a reason to introduce new --run-config ?

Reply:
Merely to explicitly differentiate between `olive-config` and `the other config`. In my opinion, think naming the other config explicitly i.e. prefixing with 'run-' adds value.

Reply:
Most people will not use advanced `--olive-config` but still `--config`. I think keeping `--config` will make Olive as simple to use as possible.

Reply:
it seems the `--config` is renamed to `--run-config`? if this is the case, we need update all readme.md for all examples.

Reply:
`--run-config` is acceptable parameter in place of `--config`. New argument it explicitly named `--package-config`.